packages/react-query/src/__tests__/useInfiniteQuery.test.tsx

```diff
@@ -846,14 +846,12 @@ describe('useInfiniteQuery', () => {
       Promise<number>,
       [QueryFunctionContext<typeof key, number>]
     >(async ({ pageParam = start, signal }) => {
-      if (signal) {
-        const onAbort = jest.fn()
-        const abortListener = jest.fn()
-        onAborts.push(onAbort)
-        abortListeners.push(abortListener)
-        signal.onabort = onAbort
-        signal.addEventListener('abort', abortListener)
-      }
+      const onAbort = jest.fn()
+      const abortListener = jest.fn()
+      onAborts.push(onAbort)
+      abortListeners.push(abortListener)
+      signal.onabort = onAbort
+      signal.addEventListener('abort', abortListener)
```

Comment:
do you know why eslint didn't report this issue? It does show up as red squiggles in my editor, but the pipeline is still green ...

![Screenshot 2023-01-20 at 13 32 36](https://user-images.githubusercontent.com/1021430/213695874-6ce9ebc9-f075-4774-8e7c-5a044c77262d.png)

Reply:
No idea, it failed for me, when i ran `test` script locally.